File: C#/uri-1009.cs
// URI - 1009

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string sellerSame = Console.ReadLine();
            double fixedSalary = double.Parse(Console.ReadLine());
            double totalCashSales = double.Parse(Console.ReadLine());

            Console.WriteLine("TOTAL = R$ " + (fixedSalary + totalCashSales * 0.15).ToString("F2"));
        }
    }
}

File: Node/uri-1009.js
// URI - 1009

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const sellerName = input

    rl.on('line', input => {
        const fixedSalary = parseFloat(input)

        rl.on('line', input => {
            const totalCashSales = parseFloat(input)

            console.log('TOTAL = R$ ' + (fixedSalary + totalCashSales * 0.15).toFixed(2))

            rl.close()
        })
    })
})